src/stealth/hiding_tricks.js

```diff
@@ -0,0 +1,350 @@
+/* eslint no-undef: 0 */
+
+
+/* istanbul ignore next */
```

Comment:
Let's add `/* istanbul ignore file */` at the beginning as we ignore all the functions here